black.toml

```diff
@@ -0,0 +1,26 @@
+
+# Example configuration for Black.
+
+# NOTE: you have to use single-quoted strings in TOML for regular expressions.
+# It's the equivalent of r-strings in Python.  Multiline strings are treated as
+# verbose regular expressions by Black.  Use [ ] to denote a significant space
+# character.
+
+[tool.black]
```

Comment:
can this be moved to `setup.cfg`
also pls add sample how to add excluding pattern

Reply:
To be honest, I don't really know, but I can try. Usually I used this file in combination with pre-commit, but I think it would be impossible if I would move it to setup.cfg

Reply:
So, I found that black will not support `setup.cfg`. Maybe I am wrong, but looks like so

Reply:
Black uses `pyproject.toml` by default. That file is [already present](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pyproject.toml) in this project, so use that instead of the non-standard `black.yaml` approach.

Reply:
Maybe switching to pyproject and poetry would be a better option in the future. A lot of authors suggest it as a new standart: https://github.com/TezRomacH/python-package-template, https://cjolowicz.github.io/posts/hypermodern-python-01-setup

Reply:
> Black uses `pyproject.toml` by default. That file is [already present](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pyproject.toml) in this project, so use that instead of the non-standard `black.yaml` approach.

Hmm, didn't see this file in repo for some reason. It would simplify it a lot